ui/component/headerCenterMenuButtons/index.js

```diff
@@ -0,0 +1,31 @@
+import { connect } from 'react-redux';
+import { doSetClientSetting } from 'redux/actions/settings';
+import { ENABLE_NO_SOURCE_CLAIMS, CHANNEL_STAKED_LEVEL_LIVESTREAM } from 'config';
+import { selectActiveChannelStakedLevel } from 'redux/selectors/app';
+import { selectClientSetting } from 'redux/selectors/settings';
+import * as SETTINGS from 'constants/settings';
+import HeaderMenuButtons from './view';
+
+const select = (state, props) => {
+  const { user } = props;
```

Comment:
I think this broke the "Go Live" button.  Where is `user` coming from -- I don't see it being passed as a prop in the new version, plus I think it usually comes from redux?